ui/desktop/electron-app/src/index.js

```diff
@@ -152,28 +82,22 @@ app.on('ready', async () => {
       'icon.png'
     );
 
-  mainWindow = new BrowserWindow(browserWindowOptions);
+  const browserWindow = new BrowserWindow(browserWindowOptions);
 
   // If the user-specified origin changes, reload the page so that
   // the CSP can be refreshed with the this source allowed
-  runtimeSettings.onOriginChange(() => mainWindow.loadURL(emberAppURL));
-
-  // If you want to open up dev tools programmatically, call
-  // mainWindow.openDevTools();
+  runtimeSettings.onOriginChange(() => win.loadURL(emberAppURL));
```

Comment:
```suggestion
  runtimeSettings.onOriginChange(() => browserWindow.loadURL(emberAppURL));
```
Should this `win` should be `browserWindow`?

Reply:
Nice catch!! Updated PR with that fix.